modules/openy_features/openy_node/config/install/core.entity_view_mode.taxonomy_term.color.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
In his PR https://github.com/ymcatwincities/openy/pull/81 taxonomy vocabulary has been moved to separate module. This view display should be moved as well.